scripts/start-story/main.go

```diff
@@ -105,6 +106,10 @@ func run() error {
 	if err != nil {
 		return errs.Wrap(err, "failed to checkout base ref, stdout:\n%s\nstderr:\n%s", stdout, stderr)
 	}
+	stdout, stderr, err = exeutils.ExecSimpleFromDir(environment.GetRootPathUnsafe(), "git", []string{"pull", "--rebase"}, nil)
```

Comment:
I might be misremembering, but can `git pull --rebase` hang due to an interactive conflict/merge process? If it cannot hang, are merge conflicts going to affect this action?

Reply:
It shouldn't since this would run in non-interactive mode. Merge conflicts would be raised as an error as the command would exit non-zero.